python/cudf/cudf/tests/test_datetime.py

```diff
@@ -2150,6 +2150,12 @@ def test_daterange_pandas_compatibility():
     assert_eq(expected, actual)
 
 
+@pytest.mark.filterwarnings("ignore:parsing timezone:DeprecationWarning")
```

Comment:
Can we use `pytest.warns` here instead? See https://docs.rapids.ai/api/cudf/stable/developer_guide/testing/#testing-code-that-throws-warnings.

Reply:
Sure thing, added